### PR TITLE
Fix Keybase verification using wrong domain for remote accounts

### PR DESCRIPTION
**Pull request by Gargron**

### CODE REVIEW

**Comment by Gargron** [comment]
cc @xgess

File: app/lib/proof_provider/keybase.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 class ProofProvider::Keybase
   BASE_URL = ENV.fetch('KEYBASE_BASE_URL', 'https://keybase.io')
-  DOMAIN = ENV.fetch('KEYBASE_DOMAIN', Rails.configuration.x.local_domain)
+  DOMAIN   = ENV.fetch('KEYBASE_DOMAIN', Rails.configuration.x.local_domain)
```
Comment (xgess) [inline app/lib/proof_provider/keybase.rb]:
```suggestion
  DOMAIN = ENV.fetch('KEYBASE_DOMAIN', Rails.configuration.x.local_domain)
```

Comment (Gargron) [reply app/lib/proof_provider/keybase.rb]: I like to align assigments :+1:

Comment (xgess) [reply app/lib/proof_provider/keybase.rb]: :cool:

**Pull request merged**